src/Extension/Resources/config/task/SprykDumpTask.yaml

```diff
@@ -0,0 +1,23 @@
+---
+id: "spryk:dump"
+short_description: "Lists all available Spryks"
+version: "0.1.0"
+command: "php %sdk_dir%/vendor/bin/spryk-dump %spryk% --level=%level%"
```

Comment:
`php %sdk_dir%/vendor/bin/spryk-dump %spryk% %level%` use this format no hard-coded options 
also you can use PRIORITY PATH resolver and default value `vendor/bin/spryk-dump`

Reply:
Fixed